Import/ReportingServicesConverter.cs

```diff
@@ -345,10 +346,13 @@ public class ReportingServicesConverter : ExternalConverterBase, IReportingServi
 
         #region TextBox control
         void ProcessTextboxControl(XElement textBoxElement, XRControl container, float yBodyOffset) {
-            IEnumerable<XElement> runs = textBoxElement.Descendants(xmlns + "TextRun");
-            if(runs.Count() > 1) {
-                ProcessTextBoxAsRichText(textBoxElement, container, yBodyOffset);
-                return;
+            List<XElement> runs = textBoxElement.Descendants(xmlns + "TextRun").ToList();
+            if(runs.Count > 1) {
+                var distinctStyles = runs.Descendants(xmlns + "Style").Select(e => e.ToString()).Distinct().Count();

```

Comment:
please use the
```cs
var distinctStylesCount = runs.Descendants(xmlns + "Style").Distinct(XNode.EqualityComparer).Count();
```
instead

Reply:
Oh, nice. Wasn't familiar with that equality comparer. That cleans it up, thanks!